app/views/categories/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Editing category</h1>
+
```

Comment:
- [ ] After signup and getting an error, If I try to again sign up I get an error of email already being used. But if I try to use the same Email and Password in Login page, it is not working. It also doesn't have any error.